source/actions3/actions/AMDetectorSetDarkCurrentValueActionInfo.cpp

```diff
@@ -0,0 +1,32 @@
+#include "AMDetectorSetDarkCurrentValueActionInfo.h"
+
+AMDetectorSetDarkCurrentValueActionInfo::AMDetectorSetDarkCurrentValueActionInfo(double newValue, const AMDetectorInfo &detectorInfo, QObject *parent) :
+    AMActionInfo3(QString(), QString(), QString(), parent)
```

Comment:
Could these have names and descriptions?  I think the earlier actionInfo's are like this too.  Generally names and descriptions are handy when debugging buggy scans in the scanActionDatabase.